<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="15" hostname="Dineshs-MacBook-Pro.local" failures="1" tests="2" name="Default test" time="12.114" errors="0" timestamp="2025-05-16T23:15:46 EDT">
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted" time="2.28"/>
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted" time="1.261">
    <failure type="java.lang.AssertionError" message="expected [SyntaxError: bad input on line 1] but found [Python is fun!]">
      <![CDATA[java.lang.AssertionError: expected [SyntaxError: bad input on line 1] but found [Python is fun!]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1590)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at testClasses.ArrayTests.SquaresofaSorted(ArrayTests.java:246)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:565)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:687)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- SquaresofaSorted -->
  <testcase classname="testClasses.ArrayTests" name="ArraysUsingList" time="0.0">
    <ignored/>
  </testcase> <!-- ArraysUsingList -->
  <testcase classname="testClasses.ArrayTests" name="MaxConsecutiveOnes" time="0.0">
    <ignored/>
  </testcase> <!-- MaxConsecutiveOnes -->
  <testcase classname="testClasses.ArrayTests" name="ArraysUsingList1" time="0.0">
    <ignored/>
  </testcase> <!-- ArraysUsingList1 -->
  <testcase classname="testClasses.ArrayTests" name="MaxConsecutiveOnes1" time="0.0">
    <ignored/>
  </testcase> <!-- MaxConsecutiveOnes1 -->
  <testcase classname="testClasses.ArrayTests" name="ArraysinPython" time="0.0">
    <ignored/>
  </testcase> <!-- ArraysinPython -->
  <testcase classname="testClasses.ArrayTests" name="SearchTheArray1" time="0.0">
    <ignored/>
  </testcase> <!-- SearchTheArray1 -->
  <testcase classname="testClasses.ArrayTests" name="SearchTheArray" time="0.0">
    <ignored/>
  </testcase> <!-- SearchTheArray -->
  <testcase classname="testClasses.ArrayTests" name="EvenNumberOfDigits1" time="0.0">
    <ignored/>
  </testcase> <!-- EvenNumberOfDigits1 -->
  <testcase classname="testClasses.ArrayTests" name="BasicOperationsinLists1" time="0.0">
    <ignored/>
  </testcase> <!-- BasicOperationsinLists1 -->
  <testcase classname="testClasses.ArrayTests" name="ApplicationsofArray1" time="0.0">
    <ignored/>
  </testcase> <!-- ApplicationsofArray1 -->
  <testcase classname="testClasses.ArrayTests" name="ApplicationsofArray" time="0.0">
    <ignored/>
  </testcase> <!-- ApplicationsofArray -->
  <testcase classname="testClasses.ArrayTests" name="SquaresofaSorted1" time="0.0">
    <ignored/>
  </testcase> <!-- SquaresofaSorted1 -->
  <testcase classname="testClasses.ArrayTests" name="EvenNumberOfDigits" time="0.0">
    <ignored/>
  </testcase> <!-- EvenNumberOfDigits -->
  <testcase classname="testClasses.ArrayTests" name="ArraysinPython1" time="0.0">
    <ignored/>
  </testcase> <!-- ArraysinPython1 -->
  <testcase classname="testClasses.ArrayTests" name="BasicOperationsinLists" time="0.0">
    <ignored/>
  </testcase> <!-- BasicOperationsinLists -->
</testsuite> <!-- Default test -->
